modules/standard/Regexp.chpl

```diff
@@ -18,12 +18,302 @@
  */
 
 /*
-   Regexp Module
+
+This module provides regular expression support.
+
+The regular expression support is built on top of the RE2 regular expression
+library. As such, the exact regular expression syntax available is the syntax
+from RE2, which is available at http://code.google.com/p/re2/wiki/Syntax and
+included here for your convenience.
+
+
+Enabling Regular Expression Support
+-----------------------------------
+
+Setting the environment variable CHPL_REGEXP to re2 will enable regular
+expression support with the RE2 library:
+
+.. code-block:: sh
+
+   export CHPL_REGEXP=re2
+
+Then, rebuild Chapel. As part of the make, the RE2 library will be
+downloaded from its Mercurial repository at re2.googlecode.com.
+
+Note: if re2 support is not enabled (which is the default), all features
+described below will compile successfully, but will result in an internal
+error at *run time*, saying "No Regexp Support".
+
+
+Using Regular Expression Support
+--------------------------------
+
+.. code-block:: chapel
+
+   use Regexp;
+   var myRegexp = compile("a+");
+
+Now you can use :proc:`regexp.search`, :proc:`regexp.match`, :proc:`regexp.split`, :proc:`regexp.matches` or :proc:`string.search`, :proc:`string.match`, :proc:`string.split`, :proc:`string.matches`, or :proc:`string.search`.
```

Comment:
When I reached the code portion, I realized that this was probably intending to match to both versions of string.search.  However, Sphinx is making them point to the same reference, so you aren't gaining anything from having the additional one here.  There may be plans in the future to allow greater specification, so you could reference both versions, but for now I'd say just remove the second match and leave a note to add it back later.